project/MimaExcludes.scala

```diff
@@ -36,6 +36,8 @@ object MimaExcludes {
 
   // Exclude rules for 3.0.x
   lazy val v30excludes = v24excludes ++ Seq(
+    // [SPARK-25362][JavaAPI] Replace Spark Optional class with Java Optional
+    ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.api.java.Optional")
```

Comment:
Sorry @mmolimar this needs a rebase; another change just added a line here and it conflicts. Otherwise looking good

Reply:
No worries. Done ;-)

Reply:
Ah, this has to have a comma after it, and the line that was moved to the end needs the comma removed